test/benchmarks/scheduling_passes.py

```diff
@@ -109,14 +109,6 @@ def setup(self, n_qubits, depth):
             dt=1e-9,
         )
         self.timed_dag = TimeUnitConversion(self.durations).run(self.dag)
```

Comment:
If you don't apply my other suggestion, `self.timed_dag` won't be used anywhere in the code (if you do apply it, it will be used to generate  `self.scheduled_dag`, so you can ignore this suggestion).

```suggestion
```

Reply:
Just added your suggestion, didn't catch that!